### DISCUSSION TITLE
Osu! Mania mobile hitbox touch input is too low

### DISCUSSION THREAD

**Discussion by Fikrienz** [ppy/osu#28471] [Ideas]
I'm a mobile player. Since osu! Lazer was updated, i like the new control for mobile in Mania, but the gameplay felt more difficult because the hitbox was too low, even my rank down so much. 
Please make the hitbox taller, And could you add an invisible hitbox so it doesn't interfere with gameplay?

**Comment by peppy** [comment]
It will be adjustable soon!

**Reply by Fikrienz** [reply]
Thank you
